acceptance_tests/features/steps/receipt.py

```diff
@@ -34,3 +35,8 @@ def _publish_object_finalize(context, case_id="0", tx_id="3d14675d-a25d-4672-a0f
                       objectId=tx_id)
 
     context.sent_to_gcp = True
+
+
+@step("the events logged for the receipted case are {expected_event_list}")
+def check_survey_launch_event_logging(context, expected_event_list):
```

Comment:
Bad function name. This step also seems to be duplicated, I would suggest writing it once in an events step file